internal/controllers/clusterextension_controller.go

```diff
@@ -273,13 +273,12 @@ func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1alp
 
 	switch unpackResult.State {
 	case rukpaksource.StatePending:
-		setStatusUnpackPending(ext, unpackResult.Message)
-		setInstalledStatusConditionUnknown(ext, "installation has not been attempted as unpack is pending")
+		setStatusInstallFalseUnpackFailed(ext, unpackResult.Message)
+		setInstalledStatusConditionInstalledFalse(ext, "installation has not been attempted as unpack is pending")
```

Comment:
Installed=False is not always correct here. If we have previously installed a bundle, and now the unpack of the next bundle is pending, we still have an installed bundle.

The installation status and the bundle unpack status are not related to each other.

Reply:
okay, just setting the unpack as failed in 0306c9db